### ISSUE TITLE
[rllib] how to add schedule for learning rate in custom loss

### ISSUE THREAD

**Issue by AprilXiaoyanLiu** [ray-project/ray#11368]
Hello, 

I am adding a custom loss by 

def custom_loss(policy_loss, loss_inputs): 

I have two questions about this custom loss, first, is the policy_loss the total loss (surrogated loss + value_function loss + entropy loss for PPO) or just policy loss? 
Second, I'm trying to add another supervised learning loss function, but I also want the learning rate of this supervised learning loss can decay with schedule (similar concept as learning rate schedule and cross entropy coefficient rate schedule, but with different schedule rate). How can I do that? I looked into the source code, and noticed for lr and entropy coefficient schedule, they are controled by setupmins in policy, how do I make that happen for custom loss?

Thanks!
April
